README.md

```diff
@@ -47,6 +47,10 @@ readme](https://www.npmjs.com/package/generic-pool).
 
 * Set `MINIFIERS_DISABLE_COMPRESSION` to 1 to disable compression.
 
+* Set `MINIFIERS_BASE_PATH` to choose a path of files served through the
+  /file endpoint. May be absolute or relative. If relative, relative to the
+  'lib' directory. Defaults to the repository root.
```

Comment:
> If relative, relative to the  'lib' directory.

In my testing, it appears to be relative to the working dir, not necessarily `lib`.

```sh
~/dev/minifiers %   MINIFIERS_BASE_PATH='.' node server.js

[...]
{ basePath: '/Users/sgomes/dev/minifiers' }
```

```sh
~/dev/minifiers/lib %   MINIFIERS_BASE_PATH='.' node ../server.js

[...]
{ basePath: '/Users/sgomes/dev/minifiers/lib' }
```

Reply:
Oh, right. I sent the default path up one directory after I realized the `tests/` directory needs to be readable, but didn't update the README.Fixed in 5fbf94c